.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: go
+
+go:
+  - "1.11.x"
+  - master
```

Comment:
develop?

Reply:
Yes, for now. 1.12.x is coming with major changes in Feb 2019. Don't know for sure what are those.